<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Иммитация</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QGroupBox" name="box_animate">
            <property name="title">
             <string>Анимация</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QPushButton" name="btn_animate_t1">
               <property name="text">
                <string>Анимировать угол тета1</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
               <property name="autoRepeatDelay">
                <number>1000000000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_animate_t2">
               <property name="text">
                <string>Анимировать угол тета2</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
               <property name="autoRepeatDelay">
                <number>1000000000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_animate_t3">
               <property name="text">
                <string>Анимировать угол тета3</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
               <property name="autoRepeatDelay">
                <number>1000000000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_animate_reset">
               <property name="text">
                <string>Сброс</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="box_static">
            <property name="title">
             <string>Статика</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Тета1</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="spn_set_t1">
               <property name="frame">
                <bool>true</bool>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::UpDownArrows</enum>
               </property>
               <property name="specialValueText">
                <string/>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-45.000000000000000</double>
               </property>
               <property name="maximum">
                <double>45.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Тета2</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="spn_set_t2">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-45.000000000000000</double>
               </property>
               <property name="maximum">
                <double>45.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Тета3</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="spn_set_t3">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-91.000000000000000</double>
               </property>
               <property name="maximum">
                <double>91.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spn_rotate"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_calc">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="box_line">
            <property name="title">
             <string>Параметры изображения</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Толщина линий</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sld_line_width">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>30</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>3</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Видимые линии</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="check_line_1">
                 <property name="text">
                  <string>1</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="check_line_2">
                 <property name="text">
                  <string>2</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="check_line_3">
                 <property name="text">
                  <string>3</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btn_save">
            <property name="text">
             <string>...</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Обработка</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_proc">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Доступные действия</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="btn_start_proc">
               <property name="text">
                <string>Подгрузить из окна &quot;Иммитация&quot;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_count_angle_1">
               <property name="text">
                <string>Подсчитать угол</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
